cpp/ql/lib/semmle/code/cpp/models/implementations/StructuredExceptionHandling.qll

```diff
@@ -1,9 +1,29 @@
 import semmle.code.cpp.models.interfaces.Throwing
 
-class WindowsDriverFunction extends ThrowingFunction {
-  WindowsDriverFunction() {
+/**
+ * The default behavior for SEH is any function may (conditionally) raise an exception.
```

Comment:
We should write out here what SEH means (i.e., Structured Exception Handling)

Reply:
Done